microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -84,6 +84,7 @@ For more information, see:
 * [Interop of native-side and web-side code](../how-to/communicate-btwn-web-native.md)
 * [Call web-side code from native-side code](../how-to/javascript.md)
 * [Call native-side code from web-side code](../how-to/hostobject.md)
+* [WebView2Script Package](/microsoft-edge/webview2/reference/index.yml) for `window.chrome.webview` JavaScript APIs
```

Comment:
is this .yml link attempting to link to
https://learn.microsoft.com/en-us/microsoft-edge/webview2/webview2-api-reference
?
Will need to change from .yml here to something else.  Continuing investigation...